components/tooltip/tooltip.vue

```diff
@@ -343,6 +352,13 @@ export default {
     }
   },
 
+  beforeCreate () {
+    if (this.$props.disabled != null && this.$props.show != null) {
+      console.warn('Tooltip: You cannot use both the disabled and show props at the same time.');
+      console.warn('The show prop will be ignored.');
+    }
+  },
```

Comment:
This is not reactive, if you change the prop after initializing it will not trigger. see `badge.vue` validateProps for an example of how to do it reactively.

Reply:
actually for vue2, i cannot even access props in beforeCreate. vue3 it works

Reply:
that's because beforeCreate doesn't exist in vue 2 lol

Reply:
-_- my bad.